docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+    # sort: 1
```

Comment:
should we remove this line and other similar lines in commited files?

Reply:
removed in all files